### PR TITLE
Updated table 

### PR DESCRIPTION
**Pull request by cemoody**
Per conversation with Southmith Chintala (@soumith) and François Chollet (@fchollet)

### CODE REVIEW

File: docs/source/comparison.rst [hunk 1]
```diff
@@ -33,7 +33,7 @@ We hope it helps you to choose an appropriate framework for the demand.
 |       +-----------------------------+-------------------+------------------------+-------------------+----------------------------------------------------+
 |       | Variable-length loops       | Y                 | Y (scan)               |                   |                                                    |
 |       +-----------------------------+-------------------+------------------------+-------------------+----------------------------------------------------+
-|       | Stateful RNNs [3]_          | Y                 |                        |                   |                                                    |
+|       | Stateful RNNs [3]_          | Y                 |                        | Y[7]_             |                                                    |
```
Comment (beam2d) [inline docs/source/comparison.rst]: `Y[7]_` -> `Y [7]_` to be parsed correctly

**Comment by beam2d** [comment]
Thank you for the fix! I added a comment on the syntax.

**Comment by cemoody** [comment]
Thanks @beam2d! I updated it again with your fix.

File: docs/source/comparison.rst [hunk 2]
```diff
@@ -33,7 +33,7 @@ We hope it helps you to choose an appropriate framework for the demand.
 |       +-----------------------------+-------------------+------------------------+-------------------+----------------------------------------------------+
 |       | Variable-length loops       | Y                 | Y (scan)               |                   |                                                    |
 |       +-----------------------------+-------------------+------------------------+-------------------+----------------------------------------------------+
-|       | Stateful RNNs [3]_          | Y                 |                        |                   |                                                    |
+|       | Stateful RNNs [3]_          | Y                 |                        | Y [7]_             |                                                    |
```
Comment (beam2d) [inline docs/source/comparison.rst]: Align the column splitter (by deleting one space after `[7]_`). Sphinx table is so inconvenient... :(

**Comment by beam2d** [comment]
Thank you, I added one more comment on your update. Sorry for inconvenience...

**Comment by cemoody** [comment]
No inconvenience at all! :) Should be fixed

**Comment by beam2d** [comment]
LGTM!

**Pull request merged**